src/test/groovy/graphql/GraphQLTest.groovy

```diff
@@ -370,7 +370,7 @@ class GraphQLTest extends Specification {
 
         then:
         result.errors.size() == 1
-        result.errors[0].class == MissingRootTypeException
+        ((ValidationError) result.errors[0]).validationErrorType == ValidationErrorType.UnknownOperation
```

Comment:
I'd like to discuss this

Previously, the same sort of problem would be caught at execution time, throwing a `MissingRootTypeException` if the operation could not be executed. Given that I've now moved forward validation, it's very unlikely the old exception will ever be triggered. However for safety I'd like to be defensive and leave in the old code. See `SchemaUtil` and `getOperationRootType`.